generator/generator.go

```diff
@@ -249,6 +254,32 @@ func NewGenerator(options Options) (*Generator, error) {
 	}, nil
 }
 
+func getLastImportPart(srcPackageImport string) string {
+	srcPackageImport = strings.Trim(srcPackageImport, " \"")
+	idx := strings.LastIndex(srcPackageImport, "/")
+	if idx < 0 {
+		return srcPackageImport
+	}
+	return srcPackageImport[idx+1:]
+}
+
+func getSrcPackageAlias(imports []*ast.ImportSpec, srcPackageImport string) string {
+	var srcPackageImportAlias string
+	for _, imp := range imports {
+		if imp.Name != nil {
+			continue
+		}
+
+		lastSrcPackagePart := getLastImportPart(srcPackageImport)
+		lastImpPart := getLastImportPart(imp.Path.Value)
+		if lastImpPart == lastSrcPackagePart {
+			srcPackageImportAlias = "__" + lastSrcPackagePart
```

Comment:
here ideally is to recursively check if there is no such import alias in project already, but i think its to much overheaded